### DISCUSSION TITLE
Better documentation of :input-value?

### DISCUSSION THREAD

**Discussion by jason44107** [vuetifyjs/vuetify#17343] [General]
Could documentation be added to say that `:input-value` can be used for one-way binding of the checked state of a `v-checkbox`, please?  I'm using Vuex, and the checked state is part of the store, so checking or unchecking the box is a mutation; hence, I want to use `:input-value` and a `@change` listener rather than using `v-model` for two-way binding.  I hope that mention of `:input-value` can be added to the [checkboxes API page](https://vuetifyjs.com/en/components/checkboxes/#api) and the [v-checkbox page](https://vuetifyjs.com/en/api/v-checkbox/)  Thank you.

**Comment by jacekkarczmarczyk** [comment]
`v-checkbox` doesn't have `input-value` property, so it can't be used for one-way binding.

**Reply by jason44107** [reply]
Are you sure about that?  It works when I try it, causing the box to be checked when the bound property is true.  And if I then misspell it (e.g. `:input-falue`), it stops working, leaving the box unchecked regardless of the bound property.

**Reply by jacekkarczmarczyk** [reply]
Are you using Vuetify 3?

**Reply by jason44107** [reply]
No, version 2.5.8.  And I now see that `:input-value` **is** documented on the [v-checkbox API page for version 2](https://v2.vuetifyjs.com/en/api/v-checkbox/).  Very well, that'll shut me up.  :)
